kernel/aster-nix/src/fs/path/mount.rs

```diff
@@ -97,6 +97,88 @@ impl MountNode {
         Ok(child_mount)
     }
 
+    /// Clone a mount node with the an root Dentry_.
+    /// The new mount node will have the same fs and parent as the original one.
+    /// But the new mount node root Dentry_ will be the given one.
+    pub fn clone_mnt(&self, root_dentry: Arc<Dentry_>) -> Arc<Self> {
+        Arc::new_cyclic(|weak_self| Self {
```

Comment:
The name `clone_mnt` seems a little bit ambiguous. "Mnt" is short for "mount". Here we can specify the function is to clone a MountNode.

Reply:
Thank you, you're right. I've changed `clone_mnt` to `clone_mount`, and I also noticed the code comment was unclear. I've updated the comment as well. Thanks again for your suggestion!

Reply:
In fact, I believe the function name should be `clone_mount_node`, as 'mount' is an action verb. This is why we use `do_mount` to denote the action of mounting. In the previous PR 'Add struct Path for VFS', we utilized `mount_node` to signify the mount point. Therefore, I suggest we maintain consistency by using similar naming conventions. Additionally, this will also harmonize with the subsequent function `copy_mount_tree`.

Reply:
Thank you for your suggestion. I agree with your point that `mount` is an action, whereas `mount_node` is a struct. They should not be confused with each other. Renaming the function to `clone_mount_node` more accurately reflects that the action of cloning is being performed on a `mount_node` struct. The verb-noun naming convention is indeed a good practice, which I have also applied to other methods within `MountNode`, such as `copy_mount_node_tree`,` detach_mount_node`, `attach_mount_node`, and `graft_mount_node_tree`. This consistency helps in maintaining clarity and predictability in the code.